pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -42,6 +44,7 @@ type bootstrapTemplateData struct {
 	PullSecret   string
 	ReleaseImage string
 	Proxy        *configv1.ProxyStatus
+	DeployImage  string
```

Comment:
Again, `RHCOSBaseURI` or similar would be more clear

You could leave a comment here that the baremetal platform is the only user currently